trade-finance-manager-ui/server/controllers/utilisation-reports/add-payment/index.ts

```diff
@@ -87,21 +99,27 @@ export const addPayment = async (req: AddPaymentRequest, res: Response) => {
     const { reportId } = req.params;
     const checkedCheckboxIds = getPremiumPaymentsCheckboxIdsFromObjectKeys(req.body);
     const feeRecordIds = checkedCheckboxIds.map((checkboxId) => getFeeRecordIdFromPremiumPaymentsCheckboxId(checkboxId));
-    const { errors, formValues, paymentNumber } = extractAddPaymentFormValuesAndValidateIfPresent(req.body);
+    const feeRecordPaymentCurrency = getFeeRecordPaymentCurrencyFromPremiumPaymentsCheckboxId(checkedCheckboxIds[0]);
+
+    const { isAddingPayment, errors, formValues, paymentNumber } = extractAddPaymentFormValuesAndValidateIfPresent(req.body, feeRecordPaymentCurrency);
+    const canSubmitFormValues = errors.errorSummary.length === 0;
```

Comment:
Might be worth just calling this something to do with errors, confused me a little with having a true/false of can we submit form values, when it feels like it's more do we have errors that we are displaying back to the user and therefore keeping data on the page

Even just formHasErrors or something